ArduCopter/heli.cpp

```diff
@@ -183,6 +182,20 @@ void Copter::heli_update_rotor_speed_targets()
                 motors.set_desired_rotor_speed(0);
             }
             break;
+        case ROTOR_CONTROL_MODE_CLOSED_LOOP_POWER_OUTPUT:
+            // pass setpoint through as desired rotor speed
+            if (rsc_control_deglitched > 0) {
+                motors.set_interlock(true);
+                if ((rpm_sensor.get_rpm(0) > 0) && ((heli_flags.using_governor_switch && heli_flags.governor_enable) || !heli_flags.using_governor_switch)){
+                    motors.set_rsc_governor_enabled(true, motors.get_gov_rpm_setpoint(), rpm_sensor.get_rpm(0));
```

Comment:
@R-Lefebvre Maybe we should not expose RPMs outside of the AP_Motors implementation, but rather request power/rotor speed in something like 0% to 100%? That way the flight-modes can request lower/higher rotor speed, and if we have an RPM sensor, the AP_MotorsHeli_RSC governor can manage output based on the AP_RPM sensor, with params defining what 0% and 100% power is in RPMs.